apps/storybook/src/WorkingGroupPage.stories.tsx

```diff
@@ -32,30 +32,25 @@ export const Normal = () => {
       firstName: 'Peter',
       lastName: 'Venkman',
       email: 'peter@ven.com',
-      role: 'Project Manager',
+      role: 'Guest' as const,
+      workingGroupRole: 'Project Manager' as const,
     },
-    name: text('Working Group name', 'Working group name'),
+    title: text('Working Group name', 'Working group name'),
```

Comment:
Changed to match the backend. (We really should have made the contract earlier.)